.travis.yml

```diff
@@ -2,12 +2,9 @@ language: ruby
 rvm:
 - 2.4.5
 - 2.5.3
-matrix:
-  allow_failures:
-  - rvm: 2.5.3
+matrix: {}
 fast_finish: true
 sudo: false
 cache: bundler
-script: "rake spec"
-install: "pushd scripts && bundle install --jobs=3 --retry=3 && popd"
-
+script: rake spec
```

Comment:
Not sure why these two lines were added? Some PR blaster crazyness?

Reply:
Looks like it just removed the quotes